tests/local_vespa/test_app/schemas/family_document.sd

```diff
@@ -10,6 +10,20 @@ schema family_document {
         stemming: none
     }
 
+    field family_name_bolding type string {
+        indexing: input family_name_index | summary | index
+        bolding: true
+    }
+
+    field family_description_bolding type string {
+        indexing: input family_description_index | summary | index
+        bolding: true
+    }
+
+    field language type string {
+        indexing: "en" | set_language
+    }
+
```

Comment:
As on the document passage, wondering if this needs to be at the top of the doc

Reply:
good catch!